src/components/general/NavigationDrawer/components/styles.less

```diff
@@ -0,0 +1,142 @@
+@import '~styles/mixins.less';
```

Comment:
Missing comfortable and compact here

Reply:
Discussed with Lotte, comfortable and compact should be the same